Src/AutoFixtureUnitTest/RandomNumericSequenceGeneratorTest.cs

```diff
@@ -108,9 +121,9 @@ public void LimitsDoesNotMatchParamsArray()
             // Fixture setup
             var unexpectedLimits = new long[]
             { 
-                Byte.MaxValue, 
-                Int16.MaxValue, 
-                Int32.MaxValue
+                (byte)10,
+                (Int16)50,
+                (Int32)20,

```

Comment:
If issue #409 is fixed then this change will be required.  In the test the Int32 is added to the constructor before the Int16.  This would violate the increasing sequence requirement.  Unless my fix to issue #409 was incorrect.